apps/climatemappedafrica/src/lib/data/blockify/explore-page.js

```diff
@@ -35,46 +38,10 @@ async function explorePage({ block: { slugs }, hurumap }) {
 
   // TODO: Move this to a PayloadCMS
   const panel = {
-    panelItems: [
-      {
-        value: "rich-data",
-        icon: "https://cms.dev.codeforafrica.org/pesayetu/wp-content/uploads/sites/2/2021/11/Group-4505.svg",
-        iconProps: {
-          src: "https://cms.dev.codeforafrica.org/pesayetu/wp-content/uploads/sites/2/2021/11/Group-4505.svg",
-          width: 44,
-          height: 44,
-          type: "svg",
-          blurDataURL:
-            "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAAECAIAAAAmkwkpAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAGElEQVR4nGNgQAP/T///f/o/jHMWiQMHACIVCyeABSwfAAAAAElFTkSuQmCC",
-          placeholder: "blur",
-        },
-      },
-      {
-        value: "pin",
-        icon: "https://cms.dev.codeforafrica.org/pesayetu/wp-content/uploads/sites/2/2022/01/Path-210-1-1.svg",
-        pin: true,
-        iconProps: {
-          src: "https://cms.dev.codeforafrica.org/pesayetu/wp-content/uploads/sites/2/2022/01/Path-210-1-1.svg",
-          width: 44,
-          height: 44,
-          type: "svg",
-          blurDataURL:
-            "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAAECAIAAAAmkwkpAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAH0lEQVR4nGNgQAP/L/z/f/r//4P/wZzT//+fBbOQAQBvnQ3r6iVM4QAAAABJRU5ErkJggg==",
-          placeholder: "blur",
-        },
-      },
-    ],
-    scrollToTopLabel: "Back To Top",
-    dataNotAvailable: "— DATA NOT AVAILABLE",
-    lazyblock: {
-      slug: "lazyblock/panel",
-    },
-    align: "",
-    anchor: "",
-    blockId: "20amuc",
-    blockUniqueClass: "lazyblock-panel-20amuc",
-    ghostkitSpacings: "",
-    ghostkitSR: "",
+    panelItems,
+    scrollToTopLabel,
+    dataNotAvailable,
+    blockId: "explore-page",
```

Comment:
1. `blockId` or `blockType`?
2. We probably should remove the `// TODO: Move this to a PayloadCMS` comment now.